doc/source/reference/apis/metadata.md

```diff
@@ -15,18 +15,23 @@ Incubating update 1:
 - we added `v1` format that better describes current SeldonMessage
 - definition through environmental variables now accepts both `yaml` and `json` input
 
+Incubating update 2:
+- we added GRPC support for both Model and Graph metadata
+- adjustments to `v1` (it is now also an array of inputs/outputs metadata) and removed explicit distinction
 
+We plan to graduate metadata features with the 1.3 release of Seldon Core.
 
 ## Examples
-- [Basic Examples for Model with Metadata](../../examples/metadata.html)
-- [SKLearn Server example with MinIO](../../examples/minio-sklearn.html)
-- [Deployment Level Metadata](../../examples/graph-metadata.html).
+- [Model Metadata (examples)](../../examples/metadata.html)
+- [Model Metadata (format examples)](../../examples/metadata_schema.html)
+- [Deployment Level Metadata](../../examples/graph-metadata.html)
+- [Metadata with GRPC](../../examples/metadata_grpc.html)
 
+- [SKLearn Server example with MinIO](../../examples/minio-sklearn.html)
```

Comment:
Probably to separate to specify "# Metadata integrations with Frameworks"

Reply:
Ok reading this back sounds confusing, what i meant is to add another section to separate the examples with that header (or similar) as title